Tuprules.tup

```diff
@@ -12,38 +12,11 @@ else
 CC = gcc
 endif
 
-ifeq (@(TUP_DEBUG),y)
-CFLAGS += -g
-else
-CFLAGS += -Os
-endif
+CFLAGS = @(CFLAGS)
 
-CFLAGS += -W
-CFLAGS += -Wall
-ifeq (@(TUP_WERROR),y)
-CFLAGS += -Werror
-endif
-CFLAGS += -Wbad-function-cast
```

Comment:
I somewhat like the idea of having different sensible configs in tree, but I'm not totally sold that a reasonable default doesn't belong in the Tupfiles. I think a reasonable default is the flags that I build with, since it is known to compile on a few platforms for at least some version of gcc.